tasks.py

```diff
@@ -27,7 +27,7 @@ def opinionated_docker_run(
     auto_remove: bool = False,
     detach: bool = True,
     environment: dict = {},
-    entrypoint: str = None,
+    entrypoint: str = "",
```

Comment:
This was a manual change of a pre-commit finding.

Reply:
2023-05-02 18:40:15 [ERROR]   Found errors in [mypy] linter!
2023-05-02 18:40:15 [ERROR]   Error code: 1. Command output:
------
tasks.py:30: error: Incompatible default for argument "entrypoint" (default has type "None", argument has type "str")  [assignment]
tasks.py:30: note: PEP 484 prohibits implicit Optional. Accordingly, mypy has changed its default to no_implicit_optional=True
tasks.py:30: note: Use https://github.com/hauntsaninja/no_implicit_optional to automatically upgrade your codebase

Reply:
Ew.  Ok.